code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -294,6 +294,64 @@
 		to_chat(usr, "<span class='info'>Can be used on its own to produce some basic items and clothing, but more can be made using the proper tools.</span>")
 
 
+/*
+ * Wax
+ */
+/obj/item/stack/sheet/wax
```

Comment:
Wax is still flammable so play add `autoignition_temperature = AUTOIGNITION_ORGANIC` here

Reply:
AUTOIGNITION_ORGANIC is 360°C, while MELTPOINT_WAX is a mere 65°C, it's usually pointless to add both melt_temperature and autoignition_temperature to the same item since one gets reached and usually destroys the item way before the other is reached or does anything meaningful.